components/NavBar.tsx

```diff
@@ -2,36 +2,42 @@ import React from 'react';
 import Image from 'next/image';
 import stytchLogo from '/public/stytch-logo.svg';
 import nextjsLogo from '/public/nextjs-logotype-dark.svg';
+import Link from 'next/link';
 
 function NavBar() {
   return (
     <div style={styles.container}>
       <div style={styles.logoContainer}>
-        <a href="https://stytch.com" rel="noopener noreferrer" target="_blank">
+        <Link href={'/'}>
           <Image alt="Stytch logo" height={20} src={stytchLogo} width={105} />
+        </Link>
+      </div>
+
+      <div style={styles.linkContainer}>
+        <a
+          className="no-underline"
+          href="https://github.com/stytchauth/stytch-nextjs-integration"
+          rel="noopener noreferrer"
+          target="_blank"
+        >
+          View on Github
         </a>
-        <span style={styles.logoFont}> + </span>
-        <a href="https://nextjs.org" rel="noopener noreferrer" target="_blank">
-          <Image alt="Next.js logo" height={40} src={nextjsLogo} width={105} />
+        <a className="no-underline" href="https://stytch.com/docs" rel="noopener noreferrer" target="_blank">
+          Stytch docs
```

Comment:
Thoughts about putting a Log in link here? Or a Sign up button?

Reply:
Will consider for future

Reply:
👍